index.html

```diff
@@ -122,6 +177,196 @@ <h2>Methods</h2>
+      an appropriate value other than "stopped". The direction attribute for extensions
+      that will not be offered by default in an initial offer MUST be initialized to
+      "stopped".
+    </p>
+    <p class="note">The list of header extensions that MUST/SHOULD be supported is listed in
+      [[RTCWEB-RTP]], section 5.2. The "mid" extension is mandatory to use when
+      BUNDLE is in use, per [[BUNDLE]] section 9.1.
+    </p>
+    <p>
+      Let [[\HeaderExtensionsNegotiated]] be an internal slot of the RTCRtpTransceiver,
+      initialized to an empty list.
+    </p>
+    <p>
+      In the algorithms for generating initial offers in [[RTCWEB-JSEP]] section 5.1.1,
+      replace "for each supported RTP header extension, an "a=extmap" line, as specified in
+      [[RFC5285]], section 5" " with "For each RTP header extension "e"
+      listed in [[\HeaderExtensionsToOffer]] where direction is not "stopped", an "a=extmap"
+      line, as specified in [[RFC5285]], section 5, with direction taken from "e"'s direction
+      attribute."
+    </p>
+    <p>
+      In the algorithm for generating subsequent offers in [[RTCWEB-JSEP]] section 5.2.2, replace "The
+      RTP header extensions MUST only include those that are present in the most recent answer"
+      with "For each RTP header extension listed in [[\HeaderExtensionsToOffer]] that is
+      also present in the most recent answer, and where direction is not "stopped", generate
+      an appropriate "a=extmap" line with "direction" set according to the rules of [[RFC5285]]
+      section 6, considering the direction in [[\HeaderExtensionsToOffer]] to indicate the
+      answerer's desired usage". This preserves the property that the set of extensions
+      may not grow with subsequent offers.
+    </p>
+    <p>
+      In the algorithm for generating initial answers in [[RTCWEB-JSEP]] section 5.3.1, replace "For
+      each supported RTP header extension that is present in the offer" with "For each
+      supported RTP header extension that is present in the offer and is also present in
+      [[\HeaderExtensionsToOffer]] with a direction different from "stopped"".
+    </p>
+    <p class="note">
+      Since JSEP does not know about WebRTC internal slots, merging this change requires
+      more work on a JSEP revision.
+    </p>
+    <p>
+      In the algorithm to <a>set an RTCSessionDescription</a>, add a bullet in the sections
+      for processing answer and pranswer (bullet 3.2.8.1.7 and 3.2.8.2.6):
+      <ul>
+        <li>Set the transceiver's [[\HeaderExtensionsNegotiated]] slot to the list of
+          RTP header extensions from the answer.
+        </li>
+      </ul>
+    </p>
+    <section>
+      <h2>Methods</h2>
+      <dl>
+        <dt>setOfferedRtpHeaderExtensions</dt>
+        <dd>
+          Execute the following steps:
+          <ol>
+            <li>Let <var>extlist</var> be the list of extensions in the argument.</li>
```

Comment:
Add a step that says that if [[\HeaderExtensionsNegotiated]] is non-empty and we are attempting to set the direction of a stopped entry that is not presently negotiated to a value other than "stopped", throw an exception.

Since we are not allowed to add to the set of negotiated extensions after the initial negotiation, we should fail rather than silently ignore that extension in the follow-up offer.